pages/blog/[postSlug].js

```diff
@@ -166,27 +213,81 @@ export default class DevblogPost extends Component {
               </div>
             </Header>
             <Container>
+              <Sidebar>
+                {post.fields.author.map(author => {
+                  return (
+                    <div>
+                      <a
+                        href={
+                          'https://twitter.com/share?url=' +
+                          window.location.href +
+                          '&amp;text=' +
+                          post.fields.title +
+                          '&amp;via=nozzle.io'
```

Comment:
I think this should just be `nozzleio`

```suggestion
                          '&amp;via=nozzleio'
```